apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx

```diff
@@ -222,136 +233,146 @@ class VisualizerModal extends React.Component {
           fullHeight
         >
           <div style={styles.modalBody}>
```

Comment:
The following section is probably easier to review [with whitespace hidden](/code-dot-org/code-dot-org/pull/34184/files?w=1)

Reply:
wow, TIL! That's so helpful